### ISSUE TITLE
Broken Pipe Error (SquareToCircle)

### ISSUE THREAD

**Issue by GrahamBenHarper** [3b1b/manim#243]
**Issue**
I've been working with setting up the library and testing the SquareToCircle, and up until this point I've been able to use already submitted issues to resolve my errors. I'm getting some strange behavior when I call ffmpeg to write the video, as shown in the screenshot underneath. It seems like ffmpeg is not being called with the correct call in scene.py, although I'm not certain. The completion sound does play when this error happens.

![manimscreenshot](https://user-images.githubusercontent.com/12531152/41325261-9dd84a20-6e76-11e8-88b4-52bbbcfad7c2.png)

**Steps I've Taken**
I've reinstalled ffmpeg and I am currently using the shell=true remedy that was suggested for line 601/602 of scene.py. I'm sure there is a problem with the interaction between ffmpeg and the library, but I'm unsure if this is due to the library or my user error. I double-checked that FFMPEG_BIN is defined correctly in constants.py, and the ffmpeg command is indeed correct.

When I run the animation with the command -ps I get a black frame (which is fine, because I haven't checked to see if my aggdraw is reverted or not), so the error only happens with rendering multiple frames.

**Comment by JoJoDeveloping** [comment]
Try figuring out how ffmpeg is called (ie replace the exec with a print), then try to figure out how it should be called to work appropiately, then apply necessary changes to your codebase. Also post your changes here so that other people don't have to deal with this.

**Comment by GrahamBenHarper** [comment]
@JoJoDeveloping I decided to do a fresh pull of the library's latest commits and try running things again.

This time I received an error stating something like `from .big_ol_pile_of_manim_imports import *
ValueError: Attempted relative import in non-package`, but I fixed that by copying the import list directly into the example_scenes.py. Everything runs just fine now.

Since the latest version of the library seems to not have any issues, I'm going to close this for now unless there are further problems with the ffmpeg functionality.
